### PR TITLE
Improved competition page

### PR DESCRIPTION
**Pull request by omichel**
This PR adds two improvements to the competition page:

1. The competition information displayed on the page and taken from the information section of the README.md file can now contain arbitrary information and supports hyperlinks.
2. The animation preview is not loaded by default. The user can click on the "Load Animation" button to load it.

The result is visible [here](https://beta.webots.cloud/run?version=R2023a&url=https://github.com/cyberbotics/wrestling/blob/main/worlds/wrestling.wbt&type=competition).

### CODE REVIEW

**Review by Jean-Eudes-le-retour — APPROVED** [review]
Very nice, everything seems to work. Cheers

File: httpdocs/js/webots-cloud.js [hunk 1]
```diff
@@ -1367,25 +1352,42 @@ ${deleteProject}`;
...
+                  replaceAll('<', '&lt;').
+                  replaceAll('>', '&gt;').
+                  replaceAll('"', '&quot;').
+                  replaceAll("'", '&#039;');
+              }
+              for (let i = information.length - 1; i >= 0; i--) {
+                const array = information[i].split(': ');
+                const name = escapeHtml(array[0].substring(2));  // skip "- "
+                const value = escapeHtml(array[1]).replace(/\[([^\]]+)\]\(([^\)]+)\)/, '<a href="$2" target="_blank">$1</a>');
+                let tr = document.createElement('tr');
```
Comment (BenjaminDeleze) [inline httpdocs/js/webots-cloud.js:1371]:
```suggestion
                const tr = document.createElement('tr');
```

File: httpdocs/js/webots-cloud.js [hunk 2]
```diff
@@ -1367,25 +1352,42 @@ ${deleteProject}`;
...
+                tr.innerHTML = `<td>${name}:</td><td style="font-weight: bold;">${value}</td>`;
+                document.getElementById('competition-information').prepend(tr);
+              }
+              // preview image
+              let div = document.createElement('div');
```
Comment (BenjaminDeleze) [inline httpdocs/js/webots-cloud.js:1376]:
```suggestion
              const div = document.createElement('div');
```

File: httpdocs/js/webots-cloud.js [hunk 3]
```diff
@@ -1367,25 +1352,42 @@ ${deleteProject}`;
...
+              div.classList.add('thumbnail-button-container');
+              let img = document.createElement('img');
```
Comment (BenjaminDeleze) [inline httpdocs/js/webots-cloud.js:1378]:
```suggestion
              const img = document.createElement('img');
```

File: httpdocs/js/webots-cloud.js [hunk 4]
```diff
@@ -1367,25 +1352,42 @@ ${deleteProject}`;
...
+              img.src = rawUrl + '/preview/thumbnail.jpg';
+              div.append(img);
+              let button = document.createElement('button');
```
Comment (BenjaminDeleze) [inline httpdocs/js/webots-cloud.js:1381]:
```suggestion
              const button = document.createElement('button');
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]

**Pull request merged**